samples/CAPI/main.c

```diff
@@ -121,7 +147,7 @@ void moveNovelChan(NrtTimestamp delta)
 }
 
 // Function to interact with Ink
-void interactWithNovelChan()
+void interactWithNovelChan(void* context)
```

Comment:
```suggestion
void InteractWithNovelChan(void* context)
```

Reply:
I ran clang-format on the entire repo and it started changing a lot of things; especially things I didn't touch. I'm not sure what our current policy is on large formatting changes like that, so I figured I'd leave the formatting changes out for now.

Reply:
Use our patch. Not sure why that's happening to you, but yeah. Patch is usually fine.

Also Windows ain't building lol.